.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Please feel free to remove whatever sections/lines in this aren't relevant. -->
+
+## Title Line Template: [Website] - [UI components]: [Brief description.]
+<!-- Use the title line as the title of your pull request, then delete these lines. 
+
+Website: Issues that impact standards.usa.gov look, feel or functionality.
+UI components: Issues that impact the look, feel or functionality of the standards themselves.
+
+-->
+
+## Description
```

Comment:
I think once these templates land on our default branch, we can iterate on them going further and break parts of the templates into categories using the `---` triple dash rulings.

```
# Bug related to [ Website ] / [ UI component ]
## Description 
---
# New component for [ Website ] / [ UI component ]
## Proposal
```

**rendered below**

# Bug related to [ Website ] / [ UI component ]
## Description 
---
# New component for [ Website ] / [ UI component ]
## Proposal